### PR TITLE
[12.0][ADD] attachment_category

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by etobella** [comment]
@rousseldenis There is some Lintint errors, can you fix it? Then we can try to unblock this PR :wink:

**Comment by rousseldenis** [comment]
@etobella :green_apple:

**Review by ivantodorovich — APPROVED** [review]

**Review by marcelsavegnago — APPROVED** [review]

File: attachment_category/README.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+==================
+Atachment Category
```
Comment (etobella) [inline attachment_category/README.rst:2]: Attachment

**Review by etobella — COMMENTED** [review]
Just a small comment (Typo error)

**Review by etobella — APPROVED** [review]

File: attachment_category/README.rst (hunk 1 above)
Comment (etobella) [reply attachment_category/README.rst:2]: @rousseldenis If you change the typo error we can proceed with the merge :wink:

Comment (rousseldenis) [reply attachment_category/README.rst:2]: @etobella I think this is not needed as ocabot will regenerate it

**Review by andreampiovesana — APPROVED** [review]
go

**Comment by rousseldenis** [comment]
@etobella

**Review by ypapouin — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**